data/ui/widgets/status.ui

```diff
@@ -279,18 +293,31 @@
                 <property name="position">0</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkBox" id="poll">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="orientation">vertical</property>
+                <property name="spacing">8</property>
+              </object>
+              <packing>
+                <property name="expand">False</property>
+                <property name="fill">True</property>
+                <property name="position">1</property>
+              </packing>
+            </child>
```

Comment:
Status widgets are quite expensive, and I don't believe that this box should be present _at all times_ even when the status isn't a poll.

The voting box widget should be injected dynamically.